bucket/eclipse-parallel.json

```diff
@@ -17,16 +17,15 @@
         ]
     ],
     "checkver": {
-        "url": "http://download.eclipse.org/technology/epp/downloads/release/release.xml",
-        "re": "\\<present\\>([\\d]{4}-[\\d]{2})\\/(?<r>[\\w]+)\\<\\/present\\>"
+        "url": "https://download.eclipse.org/technology/epp/downloads/release/release.xml",
```

Comment:
use xpath for all checkvers

https://github.com/Ash258/Extras/tree/xmls

Reply:
See https://github.com/lukesampson/scoop-extras/compare/master...Ash258:xmls

Reply:
done